.github/workflows/pathogen-repo-ci.yaml

```
@@ -238,6 +238,16 @@ jobs:
           ref: ${{ needs.workflow-context.outputs.sha }}
           path: ${{ env.NEXTSTRAIN_GITHUB_DIR }}
 
+      - name: Verify nextstrain-pathogen.yaml file
+        continue-on-error: false
+        run: >
+          # shellcheck disable=SC2154
+
+          [[ -f nextstrain-pathogen.yaml ]] || (
+               echo "To use this workflow, there must be a 'nextstrain-pathogen.yaml' file present in the repository root" \
+            && exit 1
+          )
```

_nit_

A more standard construction of this would be to use an `if` statement…

```suggestion
          if [[ ! -f nextstrain-pathogen.yaml ]]; then
               echo "To use this workflow, there must be a 'nextstrain-pathogen.yaml' file present in the repository root"
               exit 1
          fi
```